src/r2u/NOTES.md

```diff
@@ -0,0 +1,32 @@
+<!-- markdownlint-disable MD041 -->
+
+## About the base image
+
+Please check <https://github.com/devcontainers/images/tree/main/src/base-ubuntu>.
+
+This template pulls the `linux/amd64` platform image because `r2u` only supports `linux/amd64` platform.
```

Comment:
```suggestion
This template pulls the `linux/amd64` platform image because
[`r2u`](https://github.com/eddelbuettel/r2u) only supports `linux/amd64` platform.
```